pkg/search/proxy/framework/plugins/cache/cache.go

```diff
@@ -104,6 +105,15 @@ func (c *Cache) Connect(_ context.Context, request framework.ProxyRequest) (http
 	return h, nil
 }
 
+func getNegotiatedSerializer(gvk schema.GroupVersionKind) runtime.NegotiatedSerializer {
+	if scheme.Scheme.Recognizes(gvk) {
+		return scheme.Codecs.WithoutConversion()
```

Comment:
@ikaven1024 `scheme.Codecs.WithoutConversion()` should be replaced with `scheme.Codecs` here.

Reply:
> Would you describe what's the problem using `scheme.Codecs.WithoutConversion()`?

Using scheme. Codecs. WithoutConversion () watch the pods/nodes resource, will decode resource error:

```k8s.io/client-go/informers/factory.go:150: watch of *v1.Node ended with: an error on the server ("unable to decode an event from the watch stream: unable to decode watch event: Object 'Kind' is missing in '{\"metadata\":{\"name\":\"hostname\",\"selfLink\":\

```